pymc3/sampling.py

```diff
@@ -561,15 +587,27 @@ def sample(
         f'took {trace.report.t_sampling:.0f} seconds.'
     )
 
+    idata = None
+    if compute_convergence_checks or return_inferencedata:
+        if 'data.save_log_likelihood' in arviz.rcParams:
+            # if the user change the arviz default, use it
+            save_ll = arviz.rcparams['data.save_log_likelihood']
```

Comment:
3 comments on this: 
1. It should be `arviz.rcParams` like in matplotlib
2. I'll open an issue to add this rcParam to ArviZ so it gets added, it currently does not exist
3. Once implemented this if will be taken care of by `from_pymc3` so ti won't be needed, and before being implemented it cannot be triggered. I think this can actually be removed

Reply:
Okay, if I understand this correctly, the setting can't be set and you'll fix it? Can you make a 0.8.1 release then?

Reply:
With the `idata_kwargs` setting, this is now user-configurable anyways.